astropy/modeling/optimizers.py

```diff
@@ -145,8 +145,8 @@ def __call__(self, objfunc, initval, fargs, **kwargs):
         # set the values of constraints to match the requirements of fmin_slsqp
         model = fargs[0]
         pars = [getattr(model, name) for name in model.param_names]
-        bounds = [par.bounds for par in pars if par.fixed != True and
-                  par.tied == False]
+        bounds = [par.bounds for par in pars if par.fixed is not True and
```

Comment:
I think here using implicit truthiness is fine, i.e., `if not (par.fixed or par.tied)`